src/main/java/com/amuzil/omegasource/magus/skill/modifiers/data/BaseModifierData.java

```diff
@@ -0,0 +1,30 @@
+package com.amuzil.omegasource.magus.skill.modifiers.data;
+
+import com.amuzil.omegasource.magus.skill.modifiers.api.ModifierData;
+import net.minecraft.nbt.CompoundTag;
+
+public abstract class BaseModifierData extends ModifierData {
```

Comment:
Awesome- using DataTraits is perfect
We can laugh at Maht having to rewrite this in Kotlin lmao